src/main/java/seedu/address/commons/Comparators.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.commons;
+
+import java.util.Comparator;
+
+import seedu.address.model.entity.Team;
+
+/**
+ * Contains common comparators that will come in use within Alfred, for example to
+ * rank teams by score.
+ */
+public class Comparators {
+
+    /**
+     * Creates a new comparator used to sort teams by their score
+     * in descending order, mainly for the leader board.
+     *
+     * @return a new Comparator for Teams.
+     */
+    public static Comparator<Team> rankByScore() {
+        return new Comparator<Team>() {
+            @Override
+            public int compare(Team o1, Team o2) {
+                int scoreTeam1 = o1.getScore().getScore();
+                int scoreTeam2 = o2.getScore().getScore();
+                return scoreTeam2 - scoreTeam1;
```

Comment:
Our 'names', 'phone number' and other relevant fields are separate java class, for consistency, would you like to ads a Score.java? It may be easier to validate or edit 😄

Reply:
Think Score.java is there already

Reply:
Score.java is there. I used int here to make the comparison between scores easier. The first getScore() gets the team's score whereas the second on gets the integer value of the score.